<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MsgItem</class>
 <widget class="QWidget" name="MsgItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>150</width>
    <height>50</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>150</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widgetMain" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>999</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widgetMain{
	background-color: rgb(255, 255, 255);
	border-bottom: 1px solid #F2F2F2;
}

</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widgetHead" native="true">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#widgetHead{
	background-color: rgb(0, 0, 0);
	border-radius: 4px;
	border-image: url(:/icon/defaultheadpicture.jpg);
}</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lbName">
            <property name="styleSheet">
             <string notr="true">QLabel{
	background-color: transparent;
	font: 75 14pt &quot;Arial&quot;;
	color: #000000
}</string>
            </property>
            <property name="text">
             <string>Karson</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbMsg">
            <property name="styleSheet">
             <string notr="true">QLabel#labelMsg{
	background-color: transparent;
	font: 75 11pt &quot;Arial&quot;;
	color: #888888;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lbTime">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel#labelTime{
	background-color: transparent;
	font: 75 11pt &quot;Arial&quot;;
	color: #888888;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widgetCount" native="true">
            <property name="minimumSize">
             <size>
              <width>21</width>
              <height>21</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>21</width>
              <height>21</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget#widgetCount{
	border-radius: 10px;
	background-color: rgb(255, 0, 0);
}</string>
            </property>
            <widget class="QLabel" name="lbCount">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>21</width>
               <height>21</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel#labelCount{
	color: white;
	background-color: transparent;
	font: 10pt &quot;Airal&quot;;
}</string>
             </property>
             <property name="text">
              <string>99</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
